test/integration/framework/etcd.go

```diff
@@ -108,8 +108,8 @@ func startEtcd() (func(), error) {
 		GetEtcdURL(),
 		"--listen-peer-urls",
 		"http://127.0.0.1:0",
-		"--log-package-levels",
-		"*=DEBUG",
+		// "--log-package-levels",
+		// "*=DEBUG",
```

Comment:
Does this remove the verbose etcd logs from the integration tests? That would be great.

Reply:
yes, added a few more things in a standalone commit that cleans up the integration test logs significantly